index.js

```diff
@@ -33,7 +40,7 @@ function closestSemverValue(semverString, semverStringMap) {
 }
 
 const Defaults = {
-  ANDROID_VERSION: '7.0.0',
+  ANDROID_VERSION: 'Android 11',
```

Comment:
You missed this one - this either needs to be a semver, or if doing that is incorrect,  the comparison below needs to be adjusted to semversionify it before executing comparisons below —

https://github.com/pastelsky/useragent-generator/pull/4/files?file-filters%5B%5D=.js#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R113